nnvm/python/nnvm/to_relay.py

```diff
@@ -0,0 +1,530 @@
+    data = children[0]
+    pool_size = attrs.get_int_tuple('pool_size', (1, 1))
+    strides = attrs.get_int_tuple('strides', (1, 1))
+    padding = attrs.get_int_tuple('padding', (0, 0))
+    layout = attrs.get_str('layout', "NCHW")
+    ceil_mode = attrs.get_bool('ceil_mode', False)
+    count_include_pad = attrs.get_bool('layout', False)
+    return op.nn.avg_pool2d(
+        data,
+        pool_size=pool_size,
+        strides=strides,
+        padding=padding,
+        layout=layout,
+        ceil_mode=ceil_mode,
+        count_include_pad=count_include_pad)
+
+
+def _to_upsampling(children, attrs, odtype):
+    scale = attrs.get_int('scale')
+    layout = attrs.get_str('layout', 'NCHW')
+    method = attrs.get_str('method', 'NEAREST_NEIGHBOR')
+    return op.nn.upsampling(
+        children[0],
+        scale=scale,
+        layout=layout,
+        method=method)
+
+
+def _to_pad(children, attrs, odtype):
+    pad_value = attrs.get_float('pad_value', 0.0)
+    pad_width = attrs.get_tuple_tuple_int('pad_width')
+    return op.nn.pad(children[0], pad_width, pad_value=pad_value)
+
+
+def _to_exp(children, attrs, odtype):
+    return op.exp(children[0])
+
+
+def _to_log(children, attrs, odtype):
+    return op.log(children[0])
+
+
+def _to_leaky_relu(children, attrs, odtype):
+    alpha = attrs.get_float('alpha')
+    return op.nn.leaky_relu(children[0], alpha)
+
+
+def _to_full_like(children, attrs, odtype):
+    fill_value = attrs.get_float('fill_value')
+    return op.full_like(children[0], fill_value)
+
+
+def _to_greater(children, attrs, odtype):
+    out_type = attrs.get_str('out_type')
+    if out_type:
+        return op.greater(children[0], children[1]).astype(out_type)
+    else:
+        return op.greater(children[0], children[1])
+
+
+def _to_greater_equal(children, attrs, odtype):
+    out_type = attrs.get_str('out_type', None)
+    if out_type:
+        return op.greater_equal(children[0], children[1]).astype(out_type)
+    else:
+        return op.greater_equal(children[0], children[1])
+
+
+def _to_less(children, attrs, odtype):
+    out_type = attrs.get_str('out_type', None)
+    if out_type:
+        return op.less(children[0], children[1]).astype(out_type)
+    else:
+        return op.less(children[0], children[1])
+
+
+def _to_less_equal(children, attrs, odtype):
+    out_type = attrs.get_str('out_type', None)
+    if out_type:
+        return op.less_equal(children[0], children[1]).astype(out_type)
+    else:
+        return op.less_equal(children[0], children[1])
+
+
+def _to_negative(children, attrs, odtype):
+    return op.negative(children[0])
+
+
+def _to_sigmoid(children, attrs, odtype):
+    return op.sigmoid(children[0])
+
+
+def _to_strided_slice(children, attrs, odtype):
+    begin = attrs.get_int_list('begin')
+    end = attrs.get_int_list('end')
+    strides = attrs.get_int_list('strides', None)
+    return op.strided_slice(children[0], begin, end, strides=strides)
+
+
+def _to_split(children, attrs, odtype):
```

Comment:
to make the style consistent with existing converters, can we remove the ```_to``` prefix of the most conversion functions
```_split``` ?